### ISSUE TITLE
CMake scripts fail to find LuaJIT 2.1 automatically

### ISSUE THREAD

**Issue by ignatenkobrain** [minetest/minetest#4483]
It just can't find it due to `PATH_SUFFIXES` in cmake file.

I would recommend to add first check for pkg-config as

``` cmake
find_package(PkgConfig)
if (PKG_CONFIG_FOUND)
    pkg_check_modules(PC_LUAJIT QUIET luajit)
endif()
```

and then handle it properly. if pkg_config is not found then also adjust `PATH_SUFFIXES` to search in `luajit-2.1`

**Comment by Khyota** [comment]
Fedora has made a patch for this.
[http://pkgs.fedoraproject.org/cgit/rpms/minetest.git/plain/0001-use-pkg-config-to-find-luajit.patch](url)

**Comment by mark-otaris** [comment]
This is fixed by 50c33d9638c31e70e6111687b19d58ee990e744b (#7307).
